files/fr/web/html/link_types/preload/index.html

```diff
@@ -0,0 +1,222 @@
+---
+title: 'Types de liens : preload'
+slug: Web/HTML/Link_types/preload
+browser-compat: html.elements.link.rel.preload
+translation_of: 'Web/HTML/Link_types/preload'
+---
+<div>{{HTMLSidebar}}</div>
+
+<p>La valeur <code>preload</code> de l'attribut {{htmlattrxref("rel", "link")}} de l'élément {{htmlelement("link")}} permet de déclarer des requêtes à analyser dans la partie {{htmlelement("head")}} du HTML de la page, en spécifiant les ressources dont votre page va avoir besoin dans peu de temps, et qu'il serait souhaitable de charger le plus tôt possible, avant que le rendu de la page par le navigateur ne commence. Cela permet de s'assurer que les ressources sont disponibles plus tôt et qu'elles auront moins de chances de bloquer le rendu de la page, ce qui améliore les performances.</p>
```

Comment:
Instead of "requêtes à analyser", I'd use something like "des requêtes de récupération de ressource" as I think "fetch" is closer to "récupérer" and that "analyser" may be confused with "parsing"/static analysis.

Reply:
Fixed in https://github.com/mdn/translated-content/pull/1808/commits/eac2bca7098ce774dfb3371804dc2bd109b5459b